apps/web/src/app/[locale]/waitlist/_components/create.action.ts

```diff
@@ -17,6 +17,9 @@ export const sendUserSignupEmail = async (to: string) => {
     to: [to],
     subject: '🔥 Thanks for signing up to TypeHero!',
     react: UserSignupEmail(),
+    headers: {
+      'List-Unsubscribe': '<http://localhost:3000/unsubscribe>',
```

Comment:
Should this not contain similar logic as the file below? Otherwise in prod this will have unsubscribe links trying to go to `localhost:3000`

Reply:
this was just put up so tom could look at it real quick. im closing

Reply:
gotcha